src/services/models/diagnosticsModel.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This filename should be plural to be consistent with other model files.

Reply:
done